jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -523,23 +529,19 @@ private boolean isDDLCommand(int updatedCount, int columnCount) throws SQLExcept
       }
 
       if (character == '\\') {
-        antiSlash = true;
+        backslash = true;
       }
 
       if (character == '\'') {
-        if (antiSlash) {
-          antiSlash = false;
-        } else if (quoteString) {
```

Comment:
Instead of doing all of this, I think the easier fix will be to replace this `else if(quoteString) {` with `if (quoteString && sql.charAt(item - 1) != '\\') {`